daras_ai_v2/language_model.py

```
@@ -650,6 +650,26 @@ class LargeLanguageModels(Enum):
     )
 
     # https://docs.anthropic.com/claude/docs/models-overview#model-comparison
+    claude_4_sonnet = LLMSpec(
```

Pylint (3.3.7)

[convention] Class constant name "claude_4_sonnet" doesn't conform to UPPER_CASE naming style

(C0103)